packages/seacas/libraries/ioss/src/Ioss_Decomposition.h

```diff
@@ -130,6 +137,28 @@ namespace Ioss {
     bool distributionFactorConstant{false}; // T if all distribution factors the same value.
   };
 
+  IOSS_EXPORT size_t get_all_block_ioss_element_size(const std::vector<BlockDecompositionData> &blocks);
+
+  IOSS_EXPORT size_t get_all_block_ioss_offset_size(const std::vector<BlockDecompositionData> &blocks,
```

Comment:
For a specific field, the block component count is the number of components of that field on that element block. In the case of connectivity, it would be the number of nodes per element for that block topology. Same for fields that can be registered with different sizes per element block